.github/workflows/ui-tests.yml

```diff
@@ -129,3 +129,23 @@ jobs:
           path: |
             ~/Library/Logs/scan/*.log
             ./fastlane/test_output/**
+            
+  duplication-tests:
+    name: UI Tests for project with Sentry duplicated
+    runs-on: macos-15
+
+    steps:
+      - uses: actions/checkout@v4
+      - run: ./scripts/ci-select-xcode.sh "16.0"
+      - run: ./scripts/build-xcframework.sh gameOnly
```

Comment:
`m`: `gameOnly` is a bit confusing to me. Ideally this would use the artifact built here https://github.com/getsentry/sentry-cocoa/blob/166d67abd7e722717fe4cf460c20fcc28412ac1b/.github/workflows/build.yml#L86-L87

but if I'm not mistaken this workflow would need to wait until the build workflow is finished, and that would slow down all UI tests. We could add another workflow called something ui-tests-with-xcframework, and move the duplication test there.

Reply:
Let's wait to see if this will become a recurring thing, which I dont think so.
For now having the SDK compiled for the test itself works fine.